0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -221,8 +222,7 @@ are required.
 **`Promise.<null>`** A promise which resolves when the transfer has been submitted (but not necessarily accepted.)
 
 Rejects with `InvalidFieldsError` if required fields are missing from the transfer or malformed. Rejects with `DuplicateIdError` if a transfer with
-the given ID and different already exists. Rejects with `NotAcceptedError` if the transfer is rejected by the ledger due to insufficient balance or
-a nonexistant destination account.
+the given ID and different already exists. Rejects with `InsufficientBalanceError` if the transfer is rejected due to the source balance being too low. Rejects with `NotAcceptedError` if the transfer is otherwise rejected by the ledger (e.g. due to a nonexistant destination account).
```

Comment:
Might as well make the nonexistant destination account error specific as well, no? Then the connector would probably want to give an `F02: Unreachable` error.